### PR TITLE
Option to connect main pic to social profile image from facebook, github or gravatar.

### PR DESCRIPTION
**Pull request by manas94gupta**
Connect social network profile pic.

## Proposed Changes

  - Add an option to connect the main picture to social profile images from facebook, github or gravatar.
  - Updated modifying header blog post with the instructions.
  - Tested on both desktop and mobile with multiple accounts.

### CODE REVIEW

**Comment by manas94gupta** [comment]
@joariasl Please take a look at my implementation and review it. Thanks.

**Comment by joariasl** [comment]
I checked it in the code and was well. But I have this problem in my resources installed: jekyll/jemoji#57
It's necessary check it in running time.

**Comment by manas94gupta** [comment]
Okay I'll take a look.

**Comment by manas94gupta** [comment]
@joariasl I checked the issue and couldn't find a solution to that but I think that the issue is not linked to this feature. Its an issue with the jemoji plugin which can generate irrespective of this implementation.                 
Please guide me if I am wrong. Thanks.

**Comment by joariasl** [comment]
You are right. The problem it's iwth my jiemoji installation. I can't run Jekyll for that.
I will wait for the fix of the problem, because I don't have time for fix it.

**Comment by PanosSakkos** [comment]
>You are right. The problem it's iwth my jiemoji installation. I can't run Jekyll for that.
I will wait for the fix of the problem, because I don't have time for fix it.

I'm hitting the same issue, and I haven't resolved it yet..

File: _includes/header.html [hunk 1]
```diff
@@ -11,7 +11,11 @@ <h1 class="brand-heading">&nbsp;</h1>
             <span class="intro-text">{{ content }}</span>
             <h5>&nbsp;</h5>
             <a href="#{{ site.pages_list['About'] }}" class="page-scroll">
-              <img class="img-me" src="{{site.baseurl}}{{ site.me-img }}" alt="">
+              {% if site.me-img contains "https://" %}
```
Comment (eiselems) [inline _includes/header.html]: At first i was wondering what happens here ... but I think this is the most feasible way to check if the url is relative or absolute

**Comment by eiselems** [comment]
Tested it for github and facebook - worked just fine being served locally

**Comment by eiselems** [comment]
Today i learned that i can easily rebase and merge from the UI, sorry for the delay and thanks for contributing 👍

**Comment by manas94gupta** [comment]
Thanks :-)

**Pull request merged**